### PR TITLE
Implementation of the container transform animation

### PR DESCRIPTION
**Pull request by Juliana-Dimeglio**
Implementation of the container transform animation. Added changes in the ui.

https://user-images.githubusercontent.com/85245831/126002441-413037f8-e6b9-4e33-a8fe-7e3657de04c7.mp4

### CODE REVIEW

File: lib/src/ui/movie_list.dart [hunk 1]
```diff
@@ -63,66 +64,63 @@ class _MovieListState extends State<MovieList> {
...
-          return InkWell(
+      itemCount: data.results.length,
+      gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
+        crossAxisCount: NumericConstants.crossAxisCount,
+      ),
+      itemBuilder: (BuildContext context, int index) {
+        return OpenContainer(
+          closedColor: Colors.black,
+          transitionType: ContainerTransitionType.fadeThrough,
+          transitionDuration: Duration(seconds: 2),
```
Comment (MarcosCGlobant) [inline lib/src/ui/movie_list.dart]: you should make this value a constant and add a trailing comma at the end

File: lib/src/ui/movie_list.dart [hunk 2]
```diff
@@ -63,66 +64,63 @@ class _MovieListState extends State<MovieList> {
...
-                  ),
-                ),
-              );
+              openContainer();
             },
             child: GridTile(
               child: Container(
                 margin: EdgeInsets.symmetric(
-                  vertical: NumericConstants.marginImages,
+                  vertical: 7,
```
Comment (MarcosCGlobant) [inline lib/src/ui/movie_list.dart]: this value should be in the constants file

File: lib/src/ui/movie_list.dart (hunk 1 above)
Comment (Juliana-Dimeglio) [reply lib/src/ui/movie_list.dart]: Fixed

File: lib/src/ui/movie_list.dart (hunk 2 above)
Comment (Juliana-Dimeglio) [reply lib/src/ui/movie_list.dart]: Fixed

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

File: lib/src/ui/movie_list.dart [hunk 3]
```diff
@@ -63,54 +64,51 @@ class _MovieListState extends State<MovieList> {
...
-                snapshot.error.toString(),
-              );
-            }
-            return Center(
-              child: CircularProgressIndicator(),
+      body: StreamBuilder(
+        stream: widget.movieBloc.allMovies,
+        builder: (context, AsyncSnapshot<Movie> snapshot) {
+          if (snapshot.hasData) {
+            return buildList(snapshot.data!);
```
Comment (luisjavierglb) [inline lib/src/ui/movie_list.dart:71]: You have to type the StreamBuilder like `StreamBuilder<Movie>` in order to validate the type of the stream as well and the data object, if not types are assumed and thats not good.

**Review by luisjavierglb — CHANGES_REQUESTED** [review]
Minor changes requested

File: lib/src/ui/movie_list.dart (hunk 3 above)
Comment (Juliana-Dimeglio) [reply lib/src/ui/movie_list.dart:71]: Fixed

**Pull request merged**